src/main/java/org/folio/rest/impl/OaiPmhViewInstancesAPI.java

```diff
@@ -93,7 +119,7 @@ private void respondWithError(Throwable t, Handler<AsyncResult<Response>> asyncR
   }
 
   private Tuple createPostgresParams(String startDate, String endDate, boolean deletedRecordSupport,
-      boolean skipSuppressedFromDiscoveryRecords) {
+                                     boolean skipSuppressedFromDiscoveryRecords) {
```

Comment:
Please apply standard line continuation and 2 space indentation

Horizontal alignment is harder to maintain and uses more valuable horizontal space

Reply:
Done